docs/v2/spec/2018_Public_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,94 @@
+Sex of Applicant or Borrower,Integer,1 2 3 4 6,
+Sex of Co-Applicant or Co-Borrower,Integer,1 2 3 4 5 6,
+Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3,
+Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3 4,
+Age of Applicant or Borrower,Character Varying,<25 25-34 35-44 45-54 55-64 65-74 >74 8888,
+Age of Applicant >= 62,Character Varying,Yes No NA,
+Age of Co-Applicant or Co-Borrower,Character Varying,<25 25-34 35-44 45-54 55-64 65-74 >74 8888 9999,
+Age of Co-Applicant >= 62,Character Varying,Yes No NA,
+Income,Character Varying,NA,
+Type of Purchaser,Integer,0 1 2 3 4 5 6 71 72 8 9,
+Rate Spread,Character Varying,NA Exempt,
+HOEPA Status,Integer,1 2 3,
+Lien Status,Integer,1 2,
+Applicant or Borrower - Name and Version of Credit Scoring Model,Integer,1 2 3 4 5 6 7 8 9 1111,
+Co-Applicant or Co-Borrower - Name and Version of Credit Scoring Model,Integer,1 2 3 4 5 6 7 8 9 10 1111,
+Reason for Denial: 1,Integer,1 2 3 4 5 6 7 8 9 10 1111,
+Reason for Denial: 2,Integer,1 2 3 4 5 6 7 8 9,
+Reason for Denial: 3,Integer,1 2 3 4 5 6 7 8 9,
+Reason for Denial: 4,Integer,1 2 3 4 5 6 7 8 9,
+Total Loan Costs,Character Varying,NA Exempt,
+Total Points and Fees,Character Varying,NA Exempt,
+Origination Charges,Character Varying,NA Exempt,
+Discount Points,Character Varying,NA Exempt,
+Lender Credits,Character Varying,NA Exempt,
+Interest Rate,Character Varying,NA Exempt,
+Prepayment Penalty Term,Character Varying,NA Exempt,
+Debt-to-Income Ratio,Character Varying,<20% 20%-<30% 30%-<36% 50%-60% >60% Exact values between 36%-<50% NA Exempt,
+Combined Loan-to-Value Ratio,Character Varying,NA Exempt,
+Loan Term,Character Varying,NA Exempt,
+Introductory Rate Period,Character Varying,NA Exempt,
+Balloon Payment,Integer,1 2 1111,
+Interest-Only Payments,Integer,1 2 1111,
+Negative Amortization,Integer,1 2 1111,
+Other Non-Amortizing Features,Integer,1 2 1111,
+Property Value,Character Varying,NA Exempt,
+Manufactured Home Secured Property Type,Integer,1 2 3 1111,
+Manufactured Home Land Property Interest,Integer,1 2 3 4 5 1111,
+Total Units,Character Varying,1 2 3 4 5-24 25-49 50-99 100-149 >149,
+Multifamily Affordable Units,Character Varying,NA Exempt,
+Submission of Application,Integer,1 2 3 1111,
+Initially Payable to Your Institution,Integer,1 2 3 1111,
+Automated Underwriting System: 1,Integer,1 2 3 4 5 6 1111,
+Automated Underwriting System: 2,Integer,1 2 3 4 5,
+Automated Underwriting System: 3,Integer,1 2 3 4 5,
+Automated Underwriting System: 4,Integer,1 2 3 4 5,
+Automated Underwriting System: 5,Integer,1 2 3 4 5,
+Reverse Mortgage,Integer,1 2 1111,
+Open-End Line of Credit,Integer,1 2 1111,
+Business or Commercial Purpose,Integer,1 2 1111,
+Conforming Loan Limit Flag,Character Varying,C NC U NA,
+Metropolitan Statistical Area/Metropolitan Division (MSA/MD),Integer,,
+Population,Character Varying,,

```

Comment:
Please add notes to all the Census fields stating that they are from Census data.